src/main/java/eu/dissco/backend/controller/MasJobRecordController.java

```diff
@@ -0,0 +1,56 @@
+package eu.dissco.backend.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.backend.domain.AnnotationState;
+import eu.dissco.backend.domain.jsonapi.JsonApiListResponseWrapper;
+import eu.dissco.backend.domain.jsonapi.JsonApiWrapper;
+import eu.dissco.backend.exceptions.NotFoundException;
+import eu.dissco.backend.properties.ApplicationProperties;
+import eu.dissco.backend.service.MasJobRecordService;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+@RestController
+@RequestMapping("api/v1/mjr")
```

Comment:
The call I am missing is to get jobs called for a particular specimen.
Maybe this should be a call/operation on the specimen (specimens/XXX/XXX/mjr) returns all mjr's for that specimen. Maybe with a request param where you can add the status, so you can select only the ones that are scheduled or failed.
Probably best to make a new task for this.

Reply:
[Added new task](https://naturalis.atlassian.net/browse/DD-834)